elastic/kibana
# [Console] Improve text in Welcome screen

Fixes https://github.com/elastic/kibana/issues/136695


### Summary
Updates the text in the welcome panel according to the [UX doc](https://docs.google.com/document/d/1FPi3ImyXETNp1XszD8b8eR7r-6VVwEu0RGgZG5r4U2M/edit#) changes and improves visual consistency.

 ### Testing
1. Open Console in incognito mode to see the welcome panel. (You can also clear local storage to see it again.)
2. Verify that the text is consistent with the UX doc changes.


<details>

<summary>Screenshots</summary>

<img width="438" alt="Screenshot 2023-02-01 at 07 54 10" src="https://user-images.githubusercontent.com/1191206/215972603-a2723339-2620-4165-b200-599599ffe64e.png">
<img width="444" alt="Screenshot 2023-02-01 at 07 54 03" src="https://user-images.githubusercontent.com/1191206/215972610-00b6f281-c37d-49fd-a29f-adbafb13460e.png">
<img width="437" alt="Screenshot 2023-02-01 at 07 53 51" src="https://user-images.githubusercontent.com/1191206/215972616-fbb62df5-a575-4234-b69b-5bb5d2fba05b.png">


</details>
